tests/cypress/e2e/unit_tests/p1_2_fleet.spec.ts

```diff
@@ -1405,3 +1405,63 @@ describe('Test GitRepoRestrictions scenarios for GitRepo application deployment.
     })
   )
 });
+
+describe('Test Fleet `doNotDeploy: true` skips deploying resources to clusters.', { tags: '@p1_2'}, () => {
+  const key = "key_resources"
+  const value = "deploy_true"
+
+  beforeEach('Cleanup leftover Cluster labels if any.', () => {
+    cy.login();
+    cy.visit('/');
+    // Remove labels from the clusters.
+    cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+    dsAllClusterList.forEach(
+      (dsCluster) => {
+        // Adding wait to load page correctly to avoid interference with hamburger-menu.
+        cy.wait(500);
+        cy.removeClusterLabels(dsCluster, key, value);
+      }
+    )
+  })
+
+  qase(88,
+
+    it("Fleet-88: Test bundle did not get deploy when doNotDeploy value set to true option is used in the fleet.yaml.", { tags: '@fleet-88' }, () => {
```

Comment:
Consider change:
`Fleet-88: Test bundle did not get deploy when doNotDeploy value set to true option is used in the fleet.yaml` to
`Fleet-88: Test bundle did not get deployed when 'doNotDeploy' value set to true option is used in the 'fleet.yaml' file`

Reply:
Okay, I will update it.